channels/unsorted.m3u

```diff
@@ -5732,3 +5732,11 @@ https://5cf58a556f9b2.streamlock.net:443/live/tv_jejumbc/playlist.m3u8
 https://5c74939c891dc.streamlock.net/live/TV/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="http://www.etv.go.kr/images/newimg/logo.png" group-title="",한국선거방송
 http://necgokr2-724.acs.wecandeo.com/ms/2528/724/index_1.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/locomotionxtv/picture?width=300&height=300" group-title="",Locomotion TV 1
+http://locomotiontv.com/envivo/loco_ch/stream.m3u8
```

Comment:
Blocked, Cloudflare on steroids for this one.

Reply:
http://locomotiontv.com/ is blocked by Cloudflared in this moment. In Facebook post, https://www.facebook.com/locomotionxtv/photos/saludos-este-es-la-lista-m3u-que-incluye-el-canal-en-calidad-normal-y-baja-httpl/2638268926407322/, the link is operative for several days